man/systemd.network.xml

```diff
@@ -1490,6 +1490,18 @@ VRF=vrf1
 </programlisting>
     </example>
 
+    <example>
+      <title>MacVTap</title>
+      <para>This brings interface <literal>macvtap-test</literal> up and attach it to the
```

Comment:
the english isn't really in order. It should say "This brings up a network interface … and attaches it …"